### DISCUSSION TITLE
How one can run single tests?

### DISCUSSION THREAD

**Discussion by remoe** [docker-mailserver/docker-mailserver#2757] [General]
I try to run fetchmail test:

`make test/mail_fetchmail.bats`

It doesn't work because the local volume is not correctly setup:

[`-v "${PRIVATE_CONFIG}":/tmp/docker-mailserver `](https://github.com/docker-mailserver/docker-mailserver/blob/e6f481b59f7f67a7f605f9d97fa6a5ee1de68840/test/mail_fetchmail.bats#L8)

How one can run single tests?

**Comment by remoe** [comment]
If SELinux enabled then this scripts doesn't mount the local filesystem. One need to disable SELinux:

`setenforce 0`

and then run:

`make test/mail_fetchmail.bats`

```
mail_fetchmail.bats
 ✓ checking process: fetchmail (fetchmail server enabled)
 ✓ checking fetchmail: gerneral options in fetchmailrc are loaded
 ✓ checking fetchmail: fetchmail.cf is loaded
 ✓ checking restart of process: fetchmail
```

**Reply by polarathene** [reply]
Just curious, what distro are you using?

**Reply by remoe** [reply]
OS: Fedora Linux 35 (Server Edition)
Kernel: Linux 5.15.14-200.fc35.x86_64
